tmp/lanelet2_extension/docs/lanelet2_format_extension.md

```diff
@@ -269,3 +269,41 @@ _An example:_
     <tag k="area" v="yes"/>
   </way>
 ```
+
+### No Stopping Area
+
+The area with `no_stopping_area` tag can be used to prohibit even a few seconds of stopping, such as in traffic jams or at traffic lights.
+
+_An example:_
+
+```xml
+  <way id="191127">
+    <nd ref="191123"/>
+    <nd ref="191124"/>
+    <nd ref="191129"/>
+    <nd ref="191125"/>
+    <nd ref="191130"/>
+    <nd ref="191126"/>
+    <tag k="type" v="no_stopping_area"/>
+    <tag k="area" v="yes"/>
+  </way>
+```
+
+### No Parking Area
+
+The area with `no_parking_area` tag can be used to prohibit parking. Stopping for a few seconds is allowed in traffic jams or at traffic lights to this area.
+
+_An example:_
+
+```xml
+  <way id="191127">
```

Comment:
Since no_parking_area should be defined as a regulatory_element, it might be better to give an example of "relation" object rather than "way" object

Reply:
fixed in 5c8db1a7d42ff1d038e561c5dc55946ad4ee1b15